projects/plugins/jetpack/extensions/blocks/donations/amount/index.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { button as icon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+
+const name = 'donations-amount';
+const BUTTON_BEHAVIOUR_TITLE = __( 'Amount selector', 'jetpack' );
+const INPUT_BEHAVIOUR_TITLE = __( 'Custom amount field', 'jetpack' );
+
+const settings = {
+	title: __( 'Donations amount button', 'jetpack' ),
+	description: __( 'Button that determines a fixed donation amount', 'jetpack' ),
+	category: 'earn',
+	attributes: {
+		baseAmountMultiplier: {
+			type: 'string',
+		},
+		label: {
+			type: 'string',
+		},
+		currency: {
+			type: 'string',
+		},
+		amount: {
+			type: 'string',
+		},
+		disabled: {
+			type: 'boolean',
+		},
+	},
+	__experimentalLabel: ( { disabled } ) =>
+		disabled ? INPUT_BEHAVIOUR_TITLE : BUTTON_BEHAVIOUR_TITLE,
+	icon,
+	supports: {
```

Comment:
Currently I can insert a donations-amount outside of a donations block but it just errors immediately.

You could solve it by adding `inserter: false` to this property so it doesn't show up in the inserter at all.

Reply:
- [ ] Add `inserter: false`.